### ISSUE TITLE
ImageImagick doesn't provide JPEG 2000 (by default)

### ISSUE THREAD

**Issue by nemobis** [Homebrew/homebrew#45600]
As you can verify with `identify -list format | grep JP2`, the ImageMagick formula doesn't install a delegate for  JP2 and other JPEG-2000 formats.

I now see in the code that there is an option to include it, which I had missed: https://github.com/Homebrew/homebrew/blob/master/Library/Formula/imagemagick.rb#L20 (updated in https://github.com/Homebrew/homebrew/pull/38375 ). This forces one to build from source to have JP2, right?

Your call, I guess. FWIW, Ubuntu (as well as Fedora 23 and others) include JP2 by default. In any case, where could this option be documented to be easier to find?

**Comment by mistydemeo** [comment]
> In any case, where could this option be documented to be easier to find?

If you check `brew info imagemagick` or `brew options imagemagick`, it will list all of the available options.

**Comment by mikemcquaid** [comment]
These options are the same as any other formula in Homebrew. Unlike Ubuntu/Fedora we sometimes have different options to minimise dependencies on file formats that aren't as widely used.

**Comment by jm3** [comment]
@mistydemeo THANK YOU for the `brew options` trick, i never knew how to do that! that's awesome.

@nemobis ImageMagick not installing JPEG2000 handlers by default was frustrating for me too. I'm doing some OCR work with `tesseract` and `pdfbeads` which prefer to use JPEG2000 for certain page background images for better compression, and it took me a hot minute to understand how to get JPEG2000 installed.

In short:

    brew uninstall imagemagick
    brew install --force imagemagick --with-jp2

successfully did it for me. notably, contrary to instructions i found elsewhere, `brew install --force` with options does *not* rebuild or re-install the formula. it seemed like exactly what `--force` should have done, but who knows.
